docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -6,24 +6,8 @@ each of the products in your Elastic Stack. Beats and Logstash 5.6 are
 compatible with Elasticsearch {version} to give you flexibility in scheduling the
 upgrade.
 
-****
-The steps you need to take to upgrade differ depending on which products you
-are using. Want a list that's tailored to your stack? Try out our
-{upgrade_guide}[Interactive Upgrade Guide].
-****
-
-If you are running a pre-6.0 version, we recommend upgrading to the most
-recent 5.6 before upgrading to {version}. {xpack} 5.6
-provides a free Upgrade Assistant that identifies issues you need to address
-before upgrading and simplifies migrating indices that need to be reindexed
-before you upgrade. The Upgrade Assistant is enabled with both Trial and
-Basic licenses. You can install {xpack} solely for the purpose of upgrading. In
-6.3 and later, {xpack} is automatically installed when you install {es}, {kib},
-and Logstash. 
-
-Rolling upgrades are supported when upgrading from Elasticsearch 5.6 and
-Elasticsearch 6.0-6.2 to {version}. Upgrading from any
-version prior to 5.6 requires a full cluster restart.
```

Comment:
I don't think it's correct to remove this wholesale. This version covers upgrading *to* 6.7. If you're upgrading from a pre-6.3 version, the information about installing X-Pack still applies, as does using the upgrade guide (which I'm upgrading to mention 6.7). Perhaps we need to split off that last sentence in the second paragraph to make it more obvious? "In
6.3 and later, {xpack} is automatically installed when you install {es}, {kib},
and Logstash. "

Reply:
I moved it down into the body of the upgrade steps to match what you'd done in 7.0, but if you want it repeated still at the top, that's okay by me.